### PR TITLE
Improvement/zenko 3661 setup dependency mocks for replication tests

### PR DESCRIPTION
**Pull request by naren-rajendran**
Enable backbeat replication end 2 end tests.

### CODE REVIEW

File: tests/zenko_tests/e2e-config.yaml [hunk 1]
```diff
@@ -1,8 +1,25 @@
 accounts:
   - "zenko"
-endpoints: []
-locations: []
+endpoints:
+  - hostname: "http://aws-mock"
+    locationName: "awsbackendmismatch"
+  - hostname: "http://azure-mock/devstoreaccount1"
+    locationName: "azurebackendmismatch"
```
Comment (alexanderchan-scality) [inline tests/zenko_tests/e2e-config.yaml]: the endpoints seems incorrect.
`endpoints` is the list of hostnames the `s3api` can be reached by

the mock service endpoints should be created when they are deployed

**Review by nicolas2bert — APPROVED** [review]
Let's move away from the python test set up and use a JS swagger client instead. 
Let's add a comment referencing a JIRA ticket. 
This can be done after the "clean-up non-running tests" step.

**Comment by naren-rajendran** [comment]
> Let's move away from the python test set up and use a JS swagger client instead. Let's add a comment referencing a JIRA ticket. This can be done after the "clean-up non-running tests" step.

Created this ticket to address this, https://scality.atlassian.net/browse/ZENKO-3732

**Review by miniscruff — APPROVED** [review]

**Comment by naren-rajendran** [comment]
@bert-e reset

**Pull request merged**